### PR TITLE
Tomchris/setuptools

### PR DESCRIPTION
**Pull request by tomchris**
Updates #13 Implemented setuptools

Until new full devel deployment is complete install can be via 2
methods:
User:
- From virtualenv `pip install git+git://github.com/platform9/express-wizard.git@master#egg=express-wizard`
Dev:
- `git clone git://github.com/platform9/express-wizard.git`
- `cd express-wizard && pip install -e .`

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    version='0.0.1', #REQUIRED
+    description='Wizard for Platform9 express-cli and pf9-express', #OPTIONAL
+    long_description=long_description, #OPTIONAL
+    long_description_content_type='text/markdown',  #OPTIONAL
+    url='https://github.com/Platform9/express-wizard',  #OPTIONAL
+    author='Thomas Christopoulos',  #OPTIONAL
+    author_email='tom.christopoulos@platform9.com',  #OPTIONAL
+    classifiers=[  # Optional
+	'Development Status :: 3 - Alpha',
+	'Intended Audience :: Developers',
```
Comment (jb4free) [inline setup.py:38]: We should change the audience

**Review by jb4free — CHANGES_REQUESTED** [review]
minor nit

**Review by dwrightco1 — APPROVED** [review]

File: setup.py (hunk 1 above)
Comment (tomchris) [reply setup.py:38]: Added  'Intended Audience :: Information Technology',
It's a finite list used by PyPi to catalog packages.

Comment (tomchris) [reply setup.py:38]: Resolved with:
https://github.com/platform9/express-wizard/pull/19/commits/1a5a6b471772bf17a7911f619487c909aaa3962b

**Review by jb4free — APPROVED** [review]
Lgtm

**Pull request merged**